docs/content/manual/release-specific/v1.6.0/test-engine-version-enforcement.md

```diff
@@ -24,19 +25,19 @@ And should see incompatible version error in longhorn-manager Pod logs
 time="2023-08-17T03:03:20Z" level=fatal msg="Error starting manager: failed checking Engine upgarde path: incompatible Engine ei-7fa7c208 client API version: found version 7 is below required minimal version 8"
 ```
 
-**When** downgraded Longhorn to v1.5.x  
+**When** downgraded Longhorn to v1.5.x
 **Then** Longhorn components should be running
 
-**When** upgraded v1.4.1 volume (volume-1) engine  
+**When** upgraded v1.4.1 volume (volume-1) engine
 And upgraded Longhorn to v1.6.0
-**Then** Longhorn components should be running  
-And v1.4.x EngineImage state should be incompatible
+**Then** Longhorn components should be running
+And v1.4.x EngineImage state should be still deployed but status.incompatible should be true.
```

Comment:
```suggestion
And v1.4.x EngineImage state should be deployed and incompatible should be true.
```

Reply:
Modified the description and added the spaces back. Thanks.